### PR TITLE
Make brp-strip run in parallel

### PR DESCRIPTION
**Pull request by ffesti**

### CODE REVIEW

File: scripts/brp-strip [hunk 1]
```diff
@@ -5,16 +5,13 @@ if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
...
 
 # Strip ELF binaries
-for f in `find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) -exec file {} \; | \
-        grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
-	grep -v ' shared object,' | \
-	sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p'`; do
-	$STRIP -g "$f" || :
-done
+find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) | grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
+    xargs -r -P$NCPUS -n32 sh -c "file \"\$@\" | grep -v ' shared object,' | sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p' | xargs -I\{\} $STRIP -g \{\}" ARG0
```
Comment (ignatenkobrain) [inline scripts/brp-strip]: what about doing a bit less of escaping?

`sh -c 'file "$@" | grep -v " shared object," | sed -n -e "s/^\(.*\):[ \t]*ELF.*, not stripped.*/\1p" | xargs $STRIP -g' ARG0`

Comment (pmatilai) [reply scripts/brp-strip]: All these xargs-conversions will need to use find -print0 | xargs -0, see https://bugzilla.redhat.com/show_bug.cgi?id=1721348#c4

**Comment by pmatilai** [comment]
@ffesti, ping? This seems a bit forgotten...

**Comment by ffesti** [comment]
Uses -print0 now. Also replaces grep -v with `! -regex` in the find command.
Didn't change quoting as `$STRIP` needs to be taken from the current environment (yeah, I could EXPORT it...)

**Comment by pmatilai** [comment]
Doh, I've totally missed the last update to this, sorry.
With that, I don't see any reason to hold this back.

**Pull request merged**